apps/carbon/app/components/Combobox.tsx

```diff
@@ -73,40 +74,42 @@ const Combobox = forwardRef<HTMLButtonElement, ComboboxProps>(
           <PopoverContent className="min-w-[200px] w-[--radix-popover-trigger-width] p-0">
             <Command>
               <CommandInput placeholder="Search..." className="h-9" />
-              <CommandEmpty>No option found.</CommandEmpty>
-              <CommandGroup>
-                {options.map((option) => (
-                  <CommandItem
-                    value={
-                      typeof option.label === "string"
-                        ? option.label + option.helper
-                        : undefined
-                    }
-                    key={option.value}
-                    onSelect={() => {
-                      onChange?.(option.value);
-                      setOpen(false);
-                    }}
-                  >
-                    {option.helper ? (
-                      <div className="flex flex-col">
-                        <p>{option.label}</p>
-                        <p className="text-xs text-muted-foreground">
-                          {option.helper}
-                        </p>
-                      </div>
-                    ) : (
-                      option.label
-                    )}
-                    <RxCheck
-                      className={cn(
-                        "ml-auto h-4 w-4",
-                        option.value === value ? "opacity-100" : "opacity-0"
+              <ScrollArea className="overflow-auto h-96">
```

Comment:
Is this always going to be `h-96` (even when there's one or two options)?

Should we do something like this: 

```suggestion
              <ScrollArea className="overflow-auto max-h-96">
```